plugins/org.jkiss.dbeaver.ext.mysql/src/org/jkiss/dbeaver/ext/mysql/model/MySQLDialect.java

```diff
@@ -338,7 +363,8 @@ public boolean validIdentifierStart(char c) {
 
     @NotNull
     @Override
-    public String getTypeCastClause(@NotNull DBSTypedObject attribute, @NotNull String expression, boolean isInCondition) {
+    public String getTypeCastClause(@NotNull DBSTypedObject attribute, @NotNull String expression,
+            boolean isInCondition) {
```

Comment:
Could you revert this unintended change or reformat it?
If the line doesn't fit in 140 characters we format it like this
```suggestion
    public String getTypeCastClause(
        @NotNull DBSTypedObject attribute,
        @NotNull String expression,
        boolean isInCondition
    ) {
```